### ISSUE TITLE
Automatic brace-matching is annoying and how to turn it off is non-obvious

### ISSUE THREAD

**Issue by TOGoS** [emacs-php/php-mode#723]
e.g. in this situation:
```php

} else if( $something ) {
    echo "Something is {$something [CURSOR IS HERE]
}
```

When I type "}", instead of inserting a brace, the cursor just moves to the next line, as if to say "Oh don't worry small one, there is already a brace right here :)"   It is extremely annoying.  In other cases I think this mode inserts a closing brace without me typing it, which also confounds me.  I would like a mode that does syntax highlighting, auto-indentation, maybe highlight syntax errors for me, but doesn't otherwise second-guess me when I'm typing.

This might not be a a bug, per se, but after reading the README and trying to find further documentation, I have found no reference to this feature or how to turn it off.  So maybe it's a documentation issue.  Or maybe I'm using the wrong php-mode.

**Comment by yugaego** [comment]
I don't think it's `php-mode` related (I'm a user, not a maintainer.)

It looks more like `electric-pair-mode` enabled in your configuration, or something similar. Try disabling this mode or set its variable `electric-pair-skip-self` to `nil`.

**Comment by TOGoS** [comment]
You are right; turning off `electric-pair-mode` did the trick.  This is what I get for trying out a new language mode on an unfamiliar emacs install.  I'm used to having to disable `electric-indent-mode` but electric-pair-mode was new to me.
